src/main/java/com/sleekbyte/tailor/listeners/LowerCamelCaseListener.java

```diff
@@ -28,28 +28,29 @@ public LowerCamelCaseListener(Printer printer) {
     @Override
     public void enterTopLevel(TopLevelContext topLevelCtx) {
         List<IdentifierContext> names = DeclarationListener.getVariableNames(topLevelCtx);
-        names.forEach(ctx -> verifyLowerCamelCase(Messages.VARIABLE + Messages.NAMES, ctx));
+        names.forEach(ctx -> verifyLowerCamelCase(Messages.VARIABLE + Messages.NAMES, ctx, true));
     }
 
     @Override
     public void enterFunctionName(SwiftParser.FunctionNameContext ctx) {
         if (ctx.operator() == null) {
-            verifyLowerCamelCase(Messages.FUNCTION + Messages.NAMES, ctx);
+            verifyLowerCamelCase(Messages.FUNCTION + Messages.NAMES, ctx, false);
         }
     }
 
     @Override
     public void enterRawValueStyleEnumCase(SwiftParser.RawValueStyleEnumCaseContext ctx) {
-        verifyLowerCamelCase(Messages.ENUM_CASE + Messages.NAMES, ctx.enumCaseName());
+        verifyLowerCamelCase(Messages.ENUM_CASE + Messages.NAMES, ctx.enumCaseName(), false);
     }
 
     @Override
     public void enterUnionStyleEnumCase(SwiftParser.UnionStyleEnumCaseContext ctx) {
-        verifyLowerCamelCase(Messages.ENUM_CASE + Messages.NAMES, ctx.enumCaseName());
+        verifyLowerCamelCase(Messages.ENUM_CASE + Messages.NAMES, ctx.enumCaseName(), false);
     }
 
-    private void verifyLowerCamelCase(String constructType, ParserRuleContext ctx) {
-        String constructName = ctx.getText();
+    private void verifyLowerCamelCase(String constructType, ParserRuleContext ctx, Boolean unescapeIdentifier) {
```

Comment:
considering `unescapeIdentifier` is only `true` for variable and constant names, could you please make an overloaded function `verifyLowerCamelCase(constructType, ctx)` that calls this function with `unescapeIdentifier` set to `false` (so we don't have to keep passing in `false`).